src/coredata.cs

```diff
@@ -1309,21 +1750,22 @@ public partial interface NSPersistentStoreCoordinator
 		// 5.0
 		[Since(5,0)]
 		[Export ("executeRequest:withContext:error:")]
+		[return: NullAllowed]
 		NSObject ExecuteRequestwithContexterror (NSPersistentStoreRequest request, NSManagedObjectContext context, out NSError error);
```

Comment:
For the same reason I asked you not to add additional generic versions under XAMCORE_4_0. It takes too much time (to add, review and fix [1]) and we still have a lot of ground to cover on the iOS 10 API.

[1] anyway we'll have to re-audit the whole API for them later